incubator/couchdb/templates/headless.yaml

```diff
@@ -9,9 +9,9 @@ metadata:
     heritage: {{ .Release.Service }}
 spec:
   clusterIP: None
+  publishNotReadyAddresses: true
   ports:
     - name: couchdb
       port: 5984
   selector:
-    app: {{ template "couchdb.name" . }}
-    release: {{ .Release.Name }}
+{{ include "couchdb.ss.selector" . | indent 4 }}
```

Comment:
Nice cleanup on the use of a template for the StatefulSet selector here and elsewhere.